File: Perl/abc002A.pl
#!/usr/bin/perl

my $list = <STDIN>;
my @n = split(/ /, $list);
chomp($n[0]);
chomp($n[1]);
if ($n[0] < $n[1]){
  print $n[1]."\n";
}else{
  print $n[0]."\n";
}

File: JavaScript/abc002A.js
function Main(input) {
    const a = input.split(' ');
    const n = parseInt(a[0], 10);
    const m = parseInt(a[1], 10);
    if(n < m){
      console.log(m);
    }else{
      console.log(n);
    }
  }
  
Main(require("fs").readFileSync("/dev/stdin", "utf8"));